trulens_eval/examples/expositional/models/local_vs_remote_huggingface_feedback_functions.ipynb

```diff
@@ -0,0 +1,537 @@
+{
```

Comment:
<p>Can you add more details on where the model needs to be downloaded to so that the local provider can be run</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
you don't have to, it does it automatically.